app.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import FastAPI, Response
+from pydantic import BaseModel
+
+from generate_speech_service import GenerateSpeechService
+
+
+class GenerateSpeechRequest(BaseModel):
+    utterance: str
+
+
+_service = GenerateSpeechService()
+app = FastAPI()
+
+
+@app.post("/generate", response_class=Response)
```

Comment:
Let's add parent route '/api' to the backend routes? We have a SPA on the frontend and it might have it's own routing. This will help to differentiate between the two.

``
```suggestion
@app.post("/api/generate", response_class=Response)
```